webpack-babel.config.js

```diff
@@ -0,0 +1,20 @@
+var path = require('path');
+
+module.exports = {
+  output: {
+    libraryTarget: 'umd',
```

Comment:
Why UMD? The target is NodeJS here, so CommonJS (from the [`babel-plugin-webpack-loaders` examples](https://github.com/istarkov/minimal-example-for-babel-plugin-webpack-loaders/blob/c901553c80024b4e7fdf961228b0ef55de53f230/webpack.config.js#L4), this “NEEDS TO” be `commonjs2`)